apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/SamhsaMatcherTest.java

```diff
@@ -699,146 +689,6 @@ public void matchSnfClaimsByDrg() throws FHIRException {
-
-    // Set Top level diagnosis and package code to null and coding to empty
-    for (ExplanationOfBenefit.DiagnosisComponent diagnosisComponent :
-        explanationOfBenefit.getDiagnosis()) {
-      diagnosisComponent.setPackageCode(null);
-    }
-
-    // Set procedure to empty
-    explanationOfBenefit.setProcedure(new ArrayList<>());
-
-    // Set item level codings to non-SAMHSA
-    explanationOfBenefit
-        .getItem()
-        .get(0)
-        .getService()
-        .setCoding(
-            Collections.singletonList(
-                new Coding(TransformerConstants.CODING_SYSTEM_HCPCS, "H0050", "")));
-
-    // When
-    boolean isMatch = samhsaMatcher.test(explanationOfBenefit);
-
-    // Then
-    assertTrue(isMatch);
-  }
-
-  /**
-   * Verify samsha matcher for item with the given system, code and if the expectation is that there
-   * should be a match for this combination.
-   *
-   * @param system the system value
-   * @param code the code
-   * @param shouldMatch if the matcher should match on this combination
-   */
-  public void verifySamhsaMatcherForItem(String system, String code, boolean shouldMatch) {
-    SamhsaMatcher samhsaMatcher = new SamhsaMatcher();
-
-    InpatientClaim claim = getInpatientClaim();
-
-    ExplanationOfBenefit explanationOfBenefit =
-        InpatientClaimTransformer.transform(new MetricRegistry(), claim, Optional.empty());
-
-    // Set Top level diagnosis and package code to null so we can test item logic
-    for (ExplanationOfBenefit.DiagnosisComponent diagnosisComponent :
-        explanationOfBenefit.getDiagnosis()) {
-      CodeableConcept codeableConcept = diagnosisComponent.getDiagnosisCodeableConcept();
-      codeableConcept.setCoding(new ArrayList<>());
-      diagnosisComponent.setPackageCode(null);
-    }
-
-    // Set item level code to the correct coding system
-    explanationOfBenefit.getItem().get(0).getService().getCoding().get(0).setSystem(system);
-    // Set allowed CPT code
-    explanationOfBenefit.getItem().get(0).getService().getCoding().get(0).setCode(code);
-
-    assertEquals(shouldMatch, samhsaMatcher.test(explanationOfBenefit));
-  }
-
-  /**
-   * Generates the Claim object to be used in a test.
-   *
-   * @return the inpatient claim to be used for the test
-   */
-  public InpatientClaim getInpatientClaim() {
-    List<Object> parsedRecords =
-        ServerTestUtils.parseData(Arrays.asList(StaticRifResourceGroup.SAMPLE_A.getResources()));
-
-    InpatientClaim claim =
-        parsedRecords.stream()
-            .filter(r -> r instanceof InpatientClaim)
-            .map(r -> (InpatientClaim) r)
-            .findFirst()
-            .orElse(null);
-
-    if (claim != null) {
-      claim.setLastUpdated(Instant.now());
-    } else {
-      throw new IllegalStateException(
-          "Test setup issue, did not find expected InpatientClaim in sample record.");
-    }
-
-    return claim;
-  }
-
```

Comment:
Just curious: why'd all these test cases drop out in this revision?

Reply:
@cbrunefearless started porting tests a couple tests from v1 when we were first starting, and they got put in this class. Eventually to port the v1 tests we just copied the whole test file and made v1 modifications, so the tests in this class were duplicates.